pom.xml

```diff
@@ -387,6 +387,13 @@
     </dependency>
 
     <dependency>
+      <groupId>org.hamcrest</groupId>
+      <artifactId>hamcrest-all</artifactId>
+      <version>1.3</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
This is a duplicate inclusion. 

The existing hamcrest (line 407) can be updated to be restricted to the 'test' scope.